net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBGPNeighbor.xml

```diff
@@ -26,7 +26,14 @@
     <id>neighbor.updatesource</id>
     <label>Update-Source Interface</label>
     <type>select_multiple</type>
-    <help>Physical name of the interface facing the peer</help>
+    <help><![CDATA[Physical name of the IPv4 interface facing the peer. Please refer to the <a href="http://docs.frrouting.org/en/stable-7.4/bgp.html#clicmd-[no]neighborPEERupdate-source%3CIFNAME|ADDRESS%3E">FRR documentation</a> for more information.]]></help>
```

Comment:
Can we use latest instead of stable-7.4 here?

Reply:
We can, but `latest` is now `8.0` so I thought it would be better to match the actual version of FRR that is packaged with OPNsense for the best accuracy (since it's already behind `latest` and will only become more so over time).

Happy to change to `latest` if that's what's desired though.

Reply:
Indeed, let's keep it this way